File: adminWF/Classes/Attachment.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace auctionComplex.Classes
{
    [Table("attachments")]
    internal class Attachment
    {
        [Key]
        [Column("id")]
        public int Id { get; set; }

        [Column("key"), MaxLength(250)]
        public string Key { get; set; }

        [Column("name"), MaxLength(250)]
        public string Name { get; set; }

        [Column("file_size")]
        public double FileSize { get; set; }

        [Column("file_type"), MaxLength(250)]
        public string FileType { get; set; }

        [Column("timestamp")]
        public DateTime Timestamp { get; set; }

        public virtual ICollection<ChatMessage> chatMessages { get; } = new List<ChatMessage>();
    }
}

File: serverSpring/src/main/java/com/vyatsu/auctionComplex/entity/api/Attachment.java
package com.vyatsu.auctionComplex.entity.api;

import com.fasterxml.jackson.annotation.JsonIgnore;
import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

import java.time.Instant;
import java.util.LinkedHashSet;
import java.util.Set;

@Entity
@Table(name = "attachments")
public class Attachment {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    @Getter
    @Setter
    private Integer id;

    @Column(name = "key", length = 250)
    @Getter
    @Setter
    private String key;

    @Column(name = "file_size", nullable = false)
    @Getter
    @Setter
    private Double fileSize;

    @Column(name = "file_type", length = 250)
    @Getter
    @Setter
    private String fileType;

    @Column(name = "\"timestamp\"", nullable = false)
    @Getter
    @Setter
    private Instant timestamp;

    @Column(name = "name", length = 250)
    @Getter
    @Setter
    private String name;

    @OneToMany(mappedBy = "attachment")
    @JsonIgnore
    @Getter
    @Setter
    private Set<ChatMessage> chatMessages = new LinkedHashSet<>();

}